geotrellis/geotrellis-site
# First pass design

# Overview
This is a first pass at rebuilding the GeoTrellis marketing site with some minor design updates and an overhaul of the original code in order to make it easier for us (and the team) to make updates.

# Notes
- As mentioned above, there are some minor improvements (removal of transit demo from hero, more responsive site, better highlighting of current projects, usage of approved GeoTrellis branding, etc) but overall the site is close to what it is currently.
- I pulled what seemed like visually engaging screenshots for the highlighted projects (MMW, Raster Foundry, OpenTreeMap) but please let me know if there are better views! I also edited the provided text a little bit for readability / flow.
- I've added Google Tag manager to this site.

# Testing
- `git pull`
- Click through carousel, interact with inputs and ensure that they are operating as they do on geotrellis.io
- Change screen size on different page types (Homepage, Project page, Documentation page) to ensure that website looks good at various screen widths (the carousel essentially matches the current carousel's design decisions—it gives a peak of the map for visual interest but hides the interactive pane).
- Ensure that links work as expected.
- @rossbernet Would you mind reviewing the copy on all pages? I did a little reworking but definitely need an accuracy + grammar once over! Let me know if we should do this together.
- @moradology Would you mind reviewing the documentation links on the `/documentation` page?
